main_test.go

```diff
@@ -173,3 +173,15 @@ func TestAddCommand(t *testing.T) {
 		result()
 	})
 }
+
+func TestRemoveCommand(t *testing.T) {
+	t.Run("it returns no command", func(tt *testing.T) {
```

Comment:
Could we have a test for when it returns a command?

Reply:
Added additional test to check that the function is not nil.